build.gradle

```diff
@@ -25,8 +25,10 @@ repositories {
     mavenCentral() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
 }
 
-group 'com.palantir.gradle.revapi'
-version System.env.CIRCLE_TAG ?: gitVersion()
+group 'org.revapi.gradle.revapi'
```

Comment:
what about group being just `org.revapi` like the maven artifacts? or if that's not a common convention, `org.revapi.gradle`? I find the two `revapi`s just in the group id kinda weird.

Reply:
makes sense, I've udpated this to `org.revapi`